README.md

```diff
@@ -1,18 +1,27 @@
-# Краткая инструкция по работе
+# Work instructions 👩‍🎓👨‍🎓
+
+## To get started, must be installed: 🔔
```

Comment:
1. Replace with "Getting Started"
2. Emoji should be written with shortcode (https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md)
3. Add two sections to it (h3) : "Prerequisites" and "Installation"
    - "Prerequisites" should be the list of technologies / tools (node/gulp/npm)
    - "Installation" should describe how to get the project in a local environment (1/ Clone the repo, 2/ Install NPM packages) (see https://github.com/othneildrew/Best-README-Template#installation for example)